### PR TITLE
TT-17 Implemented UI component for Story creation

### PR DESCRIPTION
**Pull request by tomirisk**
Please review my UI component for story creation. Will appreciate any suggestions how to make story creation component prettier. Thank you!

### CODE REVIEW

File: src/components/stories/index.js [hunk 1]
```diff
@@ -11,9 +13,54 @@ const Stories = () => {
   const privateStories = stories.filter((s) => s.visibility === "PRIVATE");
   const publicStories = stories.filter((s) => s.visibility === "PUBLIC");
 
+  const [image, setImage] = useState(null);
+  const [story, setStory] = useState(null);
+
+  const create = () => {
+    storyService.createStory("me", story, image);
```
Comment (sghosalkar) [inline src/components/stories/index.js]: If there is no image selected, then we shouldn't allow story creation. We can either add a validation here to not create the story if `image == null` or we can disable the `Create Story` button if `image == null`

File: src/components/stories/index.js [hunk 2]
```diff
@@ -11,9 +13,54 @@ const Stories = () => {
...
+                  </label>
+                  <input id="fileInput" type="file"
+                         onChange={(event) =>
+                             setImage(event.target.files[0])} />
+                </div>
+          }
+        </div>
+
+        <div className="mt-2 align-self-center margin-left-8">
+          <a className="btn btn-primary btn-sm rounded-pill fa-pull-right text-white
```
Comment (sghosalkar) [inline src/components/stories/index.js]: To disable this button if image is null, you can add this code `${image ? '' : 'disabled'}`

**Review by sghosalkar — APPROVED** [review]

**Review by galen94 — APPROVED** [review]
If the changes that Shubham mentioned have been addressed, then I don't have any other comments. Great job!

File: src/components/stories/index.js (hunk 1 above)
Comment (sghosalkar) [inline src/components/stories/index.js]: We should not allow story creation if there is no logged in user

File: src/components/stories/index.js (hunk 2 above)
Comment (tomirisk) [reply src/components/stories/index.js]: Thank you! I have disabled the button the way you have suggested.

File: src/components/stories/index.js (hunk 1 above)
Comment (tomirisk) [reply src/components/stories/index.js]: It is disabled when there is no image selected

**Pull request merged**